pyproject.toml

```diff
@@ -3,7 +3,7 @@ omit = ["*/tests/*", "*pygmt/__init__.py"]
 
 [tool.pytest.ini_options]
 minversion = "6.0"
-addopts = "--verbose --durations=0 --durations-min=0.2 --doctest-modules --mpl --mpl-results-path=results"
+addopts = "--verbose --durations=0 --durations-min=0.2 --doctest-plus --mpl --mpl-results-path=results"
```

Comment:
> For problem 3, there is still no easy way, but I think it's possible to comment the __doctest_skip__ line, run the tests, and then revert the changes.

Actually, from reading the doctest-plus docs (https://github.com/astropy/pytest-doctestplus/tree/v0.12.0#setup-and-configuration), it says:

> Using pytest's built-in `--doctest-modules` option will override the behavior of this plugin, even if `doctest_plus = enabled in setup.cfg`, and will cause the default doctest plugin to be used...

So maybe we just need to edit the Makefile, and have have a `make doctest` command that uses `--doctest-modules` instead of `doctest-plus` to run all of these doctests? Could also setup a CI to have it run weekly if needed.

Reply:
Good suggestion. I removed the `--doctest-plus` option from `pyproject.toml`, and added it to `make test` command. I also the `make fulltest` target with the `--doctest-modules` option (I think `make doctest` is misleading).

Now, running `make test` collects 541 tests, and running `make fulltest` collects 542 tests. The extra one in `make fulltest` comes is the `grdlandmask` doctest.